### DISCUSSION TITLE
klog for vim

### DISCUSSION THREAD

**Discussion by 73** [jotaen/klog#194] [Show and tell]
I made a simple klog vim syntax theme: https://github.com/73/vim-klog
enjoy.

**Comment by jotaen** [comment]
Oh cool, thanks for making this 🎉 

If you want, I’m happy to link this from [the main website](https://klog.jotaen.net/#editor-support), so that other users are able to find it easily.

**Reply by 73** [reply]
sure. 😊

**Reply by jotaen** [reply]
Done!
